.bundlewatch.config.json

```diff
@@ -10,7 +10,7 @@
     },
     {
       "path": "./dist/css/ouds-web-reboot.css",
-      "maxSize": "4.5 kB"
+      "maxSize": "4.75 kB"
```

Comment:
Is that only because of the header ? That's unlucky

Reply:
Yes 🤦